### PR TITLE
Create page in AMPLIFY Unified Catalog “authenticate-an-api-secured-with-oauth-published-from-an-external-gateway”

### PR DESCRIPTION
**Pull request by tsjohns9**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by tsjohns9** [comment]
Can someone please add @robertengaxway as a review for this. It doesn't look like I am able to do so.

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Authenticate an API secured with OAuth and published from an external gateway
```
Comment (andreamussap) [inline content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:2]: > Title: 'Authenticate an API secured with OAuth and published from an external gateway'

long headings clutter the menu. Can't you make this short? What's an API published from an external gateway, an external API? :) Could you say 'Authenticate an external API secured with OAuth', or just 'Authenticate an API secured with OAuth' and in the text explain that this API was published from an external gtw?

cc @robertengaxway

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+linkTitle: Authenticate an API secured with OAuth
+weight: 30
+date: 2020-04-28
+description: Learn how to authenticate an API secured with OAuth credentials, which was published from an external gateway and .
+---
+The Unified Catalog has the ability to test API methods for OAS2 and OAS3 APIs. The current support for testing APIs published from an external gateway from the browser includes APIs secured with an API key, JWT authentication, and no authentication at all. At this time, there is no support for OAuth from the catalog in the browser. You may still publish your APIs to the Unified Catalog if your API is secured with OAuth. We recommend you use a tool like Postman for testing your APIs. To learn more, see [Authorizing requests](https://learning.postman.com/docs/postman/sending-api-requests/authorization/#oauth-20).
```
Comment (andreamussap) [inline content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:8]: > from the browser

This is not clear, do you mean from inside the Catalog, when you click to see the details of an API?

Comment (andreamussap) [inline content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:8]: > there is no support for OAuth from the catalog in the browser/ You may still publish your APIs to the Unified Catalog if your API is secured with OAuth

 Either it's possible of not,

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+![Postman edit auth](/Images/central/catalog/postman-edit-auth.png)
+
+After selecting **OAuth 2.0**, click the **Get New Access Token** button that is now available on the right.
+
+![Postman get token button](/Images/central/catalog/postman-get-token-button.png)
+
+This will open another dialog box with authorization details to retrieve the OAuth token. The details displayed in this dialog box will depend on the API you have downloaded, and will not reflect every OAuth use case.
+
+Set the Callback URL to <https://www.postman.com/oauth2/callback>, and list this value as an acceptable callback URL defined within the application you are using to perform the OAuth login.
```
Comment (andreamussap) [inline content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:42]: > and list this value as an acceptable callback URL defined within the application you are using to perform the OAuth login

Will the user know where's this application, is it in the Catalog or in Postman? I wonder if this is clear to the user

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+Click **Update** to complete the authorization process.
+
+![Postman update collection authentication](/Images/central/catalog/postman-update-collection-auth.png)
+
+After authenticating and setting the OAuth token, expand the collection and select a method. Once you select a method, click the **Authorization** tab,  and from the **Type** dropdown list select **Inherit auth from parent** to use the new OAuth token.
+
+![Postman select method](/Images/central/catalog/postman-select-method.png)
+
+After setting the authentication you can try the method by clicking **Send**.
```
Comment (andreamussap) [inline content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:64]: and then you don't need to go back to the Catalog to verify anything, that your external API is now authenticated?

**Comment by andreamussap** [comment]
@robertengaxway On overall, I think that this page doesn't provide a good user experience because it's a bit confusing to follow with many screenshots and text between them. 

The procedure would be easier to follow and accomplish if there were a list of numbered steps, followed by a video demonstrating how to execute them. This is a common feedback that we have from customers: having more videos showing how to perform tasks, specially in this case when you cross over different applications.

However, you don't need to address this now. Once the page is ready we can publish it, and you can edit it later to remove the screenshots and add a video if you like.

**Comment by andreamussap** [comment]
@tsjohns9 @robertengaxway I'm waiting for you to "Resolve conversations" so I can finish the review and approve this PR.

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md]: The Catalog has API and Test Methods, just like Central.   However, what we used to do until now was only Test APIs which were managed by the Axway Cloud, Service Mesh, V7 Gateways or AWS Gateways.
The point of this document is to describe that we can now Test APIs we dont manage using our gateways.   Hence the title including "external gateway".

Comment (robertengaxway) [reply content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md]: No this is not possible for APIs in an external gateway.   This is why we are writing this doc to handle the use case of publishing in the Catalog, APIs we do not manage with our gateways.

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:45]: The application is technically not in the Catalog, its somewhere else and presumably calls the APIs.   We are using the Catalog test methods to call the APIs represented in the Catalog.   The exception is that OAuth has to be tested with Postman (not in the Catalog).

File: content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md (hunk 4 above)
Comment (robertengaxway) [reply content/en/docs/central/catalog/authenticate-an-api-secured-with-oauth-published-from-an-external-gateway.md:67]: The goal was to provide instructions to use Postman for OAuth testing of the external gateway API example.   So once you get the specification from the Catalog, you use Postman to complete the verification/testing of the API.   No need to return to the Catalog

**Comment by andreamussap** [comment]
@robertengaxway This is ready to be merged but is affected by the [markdownlint issue](https://github.com/DavidAnson/markdownlint/issues/286). We are going to wait and see if the issue can be fixed easily, otherwise we'll temporarily disable the rogue rule and merge this tomorrow morning.

**Pull request merged**